src/action/wallet.js

```diff
@@ -337,6 +349,7 @@ class WalletAction {
   initPrevRestorePage() {
     if (this._store.wallet.restoreIndex >= 3) {
       this._store.wallet.restoreIndex -= 3;
+      this._store.wallet.focusedRestoreInd = this._store.wallet.restoreIndex;
```

Comment:
If `store.wallet.focusedRestoreInd` can be computed from `this._store.wallet.restoreIndex` why is it stored twice?

Reply:
It can't be computed from `restoreIndex` all the time, because it's set [here](https://github.com/lightninglabs/lightning-app/pull/1106/files#diff-c52251eee698fdb351cf7e92c36e4136R79) and used to `autofocus` the next input when the user hits `Return` during restore. 

I tried to combine them but it makes [this](https://github.com/lightninglabs/lightning-app/pull/1106/files#diff-fbb9a2d7e2e2b04543043634746adfe1R334) logic complicated :( `restoreIndex` always needs to be the index of the first input of the screen, because it's [used](https://github.com/lightninglabs/lightning-app/blob/master/src/computed/seed.js#L23) to compute the current indexes to display.

Kind of hard to explain, I can send demo vids if this doesn't make sense.